src/comparator/enum_comparator.py

```diff
@@ -3,42 +3,62 @@
+        # 1. If original EnumDescriptor is None, then a new
+        # EnumDescriptor is added.
         if self.enum_original is None:
-           msg = 'A new Enum {} is added.'.format(self.enum_update.name)
-           FindingContainer.addFinding(FindingCategory.ENUM_ADDITION, "", msg, False)
+            msg = 'A new Enum {} is added.'.format(self.enum_update.name)
+            FindingContainer.addFinding(
+                FindingCategory.ENUM_ADDITION, "", msg, False)
 
-        # 2. If updated EnumDescriptor is None, then the original EnumDescriptor is removed.
+        # 2. If updated EnumDescriptor is None, then the original
+        # EnumDescriptor is removed.
         elif self.enum_update is None:
-           msg = 'An Enum {} is removed'.format(self.enum_original.name)
-           FindingContainer.addFinding(FindingCategory.ENUM_REMOVAL, "", msg, True)
-           
-        # 3. If both EnumDescriptors are existing, check if the name is changed.
+            msg = 'An Enum {} is removed'.format(self.enum_original.name)
+            FindingContainer.addFinding(
+                FindingCategory.ENUM_REMOVAL, "", msg, True)
+
+        # 3. If both EnumDescriptors are existing, check if the name
+        # is changed.
         elif self.enum_original.name != self.enum_update.name:
-           msg = 'Name of the Enum is changed, the original is {}, but the updated is {}'.format(self.enum_original.name, self.enum_update.name)
-           FindingContainer.addFinding(FindingCategory.ENUM_NAME_CHANGE, "", msg, True)
+            msg = 'Name of the Enum is changed, the original is' \
+                '{self.enum_original.name}, but the updated is ' \
+                '{self.enum_update.name}'
+            FindingContainer.addFinding(
+                FindingCategory.ENUM_NAME_CHANGE, "", msg, True)
 
-        # 4. If the EnumDescriptors have the same name, check the values of them stay the same.
-        # Enum values are identified by number, not by name.
+        # 4. If the EnumDescriptors have the same name, check the values
+        # of them stay the same. Enum values are identified by number,
+        # not by name.
         else:
-            enum_values_dict_original = {x.number: x for x in self.enum_original.values}
-            enum_values_dict_update = {x.number: x for x in self.enum_update.values}
+            enum_values_dict_original = {
+                x.number: x for x in self.enum_original.value}
+            enum_values_dict_update = {
+                x.number: x for x in self.enum_update.value}
             # Compare Enum values that only exist in original version
-            for number in list(set(enum_values_dict_original.keys()) - set(enum_values_dict_update.keys())):
-                EnumValueComparator(enum_values_dict_original[number], None).compare()
+            for number in list(set(enum_values_dict_original.keys()) -
+                               set(enum_values_dict_update.keys())):
```

Comment:
Nit: outside the scope of this review (which mostly seems to be formatting), but we can iterate over sets directly without having to turn them into lists (and we also don't need to call `keys`; keys is the implicit iteration over a dictionary):
```python
for n in set(enum_values_dict_original) - set(enum_values_dict_update):
```

I'm not going to tag all the instances, but the nit applies to all `list(set())` constructions.

Reply:
You are right! Added this change in this #2 PR. Thanks!